lib/core/device.js

```diff
@@ -34,7 +34,7 @@ var insuletOmniPod = require('../drivers/insuletDriver');
 var oneTouchUltra2 = require('../drivers/oneTouchUltra2');
 var abbottFreeStyleLite = require('../drivers/abbottFreeStyleLite');
 var bayerContourNext = require('../drivers/bayerContourNext');
-var animasPing = require('../drivers/animasDriver');
+var animasDriver = require('../drivers/animasDriver');
```

Comment:
Yes, both will use the same driver, so I needed to make the driver name more generic.